apps/test/integration/levelSolutions/applab/ec_design.js

```diff
@@ -1187,9 +1187,10 @@ module.exports = {
         dragElement(button[0], 350, 0);
         assert.equal(designModeViz.find('#design_button1').length, 0, "button was deleted");
 
+        // TODO(dave): re-enable this after we move to headless chrome
```

Comment:
Might be worth at least clarifying the comment to make it clear we don't know about the chrome behavior.